src/modules/basket/components/BasketSummaryNav.tsx

```diff
@@ -0,0 +1,132 @@
+import styled from "@emotion/styled";
+import { PaperPlaneRight } from "@phosphor-icons/react";
+import { Button, ConfigProvider, Modal, Result, Typography, theme } from "antd";
+import { useState } from "react";
+type BasketSummaryNavProps = {
+  totalPrice: number;
+};
+
+const BasketSummaryNav: React.FC<BasketSummaryNavProps> = ({ totalPrice }) => {
+  const [isModalOpen, setIsModalOpen] = useState(false);
+  const [modalStatus, setModalStatus] = useState<
+    "idle" | "pending" | "success" | "error"
+  >("idle");
+  const {
+    token: { colorPrimary, colorBgBase },
+  } = theme.useToken();
+  const openModal = () => {
+    setModalStatus("idle");
+    setIsModalOpen(true);
+  };
+  const handleOk = () => {
```

Comment:
Optional: should we change it name? like `handleModalSubmit`, 'handleSendOrder`

Reply:
✅Done! Change to handleSubmitOrder